index.html

```diff
@@ -1,726 +1,506 @@
-            </div>
-            <div class="offer-details">
-                <p class="offer-title">Wilderness Club at Big Cedar</p>
-                <p class="offer-date-price">28 October — 1 November</p>
-                <span class="price">
-                    <span class="price-large">$2016</span> 
-                    <span class="price-small">/ 6 nights</span>
-                </span>
-            </div>
-        </div>
-
-        <!-- second deal -->
-        <div class="special-offer">
-            <img src="hotel images/Rectangle 9 (2).png" alt="Offer Image" class="offer-image">
-            <div class="rate">
-                <div class="rating-container">
-                    <img src="hotel images/Rectangle 19.png" alt="Rating Background" class="rating-background">
-                    <div class="rating-content">
-                        <img src="hotel images/Star 1.png" alt="Star Icon" class="star-icon">
-                        <span class="rating-text">4.8</span>
-                    </div>
-                </div>
-            </div>
-            <div class="heart">
-                <img src="hotel images/favorite (1).svg" alt="Heart Icon" class="heart-icon">
-            </div>
-            <div class="offer-details">
-                <p class="offer-title">Wilderness Club at Big Cedar</p>
-                <p class="offer-date-price">28 October — 1 November</p>
-                <span class="price">
-                    <span class="price-large">$2016</span> 
-                    <span class="price-small">/ 6 nights</span>
-                </span>
-            </div>
-        </div>
-                <!-- third deal -->
-        <div class="special-offer">
-            <img src="hotel images/Rectangle 9 (3).png" alt="Offer Image" class="offer-image">
-            <div class="rate">
-                <div class="rating-container">
-                    <img src="hotel images/Rectangle 19.png" alt="Rating Background" class="rating-background">
-                    <div class="rating-content">
-                        <img src="hotel images/Star 1.png" alt="Star Icon" class="star-icon">
-                        <span class="rating-text">4.8</span>
-                    </div>
-                </div>
-            </div>
-            <div class="heart">
-                <img src="hotel images/favorite (1).svg" alt="Heart Icon" class="heart-icon">
-            </div>
-            <div class="offer-details">
-                <p class="offer-title">Wilderness Club at Big Cedar</p>
-                <p class="offer-date-price">28 October — 1 November</p>
-                <span class="price">
-                    <span class="price-large">$2016</span> 
-                    <span class="price-small">/ 6 nights</span>
-                </span>
-            </div>
-        </div>
-                <!-- fourth deal -->
-        <div class="special-offer">

```

Comment:
Please remove all html used for the rendered sections just leave the ID name used to call the rendered sections in javascript